dataminer/Functions/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Maps.md

```diff
@@ -4,17 +4,140 @@ uid: DashboardMaps
 
 # Maps
 
-This component is used to display markers and/or lines on a map. It uses one or more GQI queries as data input.
+Available from DataMiner 10.4.0 [CU13]/10.5.0 [CU1]/DataMiner 10.5.4 onwards<!-- RN 42309 -->.
 
-The Maps component is fully available from DataMiner 10.4.0 [CU13]/10.5.0 [CU1]/DataMiner 10.5.4 onwards.<!-- RN 42309 --> Prior to this, it is available in preview, if the [soft-launch option](xref:SoftLaunchOptions) *ReportsAndDashboardsGQIMaps* is enabled.
+The maps component lets you **visualize and explore geospatial data using markers and/or lines**. It currently only supports Google Maps ("gmaps") as the [Maps provider](xref:Configuring_the_DataMiner_Maps_host_servers).
 
 ![Maps](~/dataminer/images/Maps_Component.png)<br>*Maps component in DataMiner 10.4.4*
 
-> [!NOTE]
->
-> - To display the map, the machine running the browser must have internet access.
-> - To use the Maps component, the host servers for DataMiner Maps have to be configured in the file `C:\Skyline DataMiner\Maps\ServerConfig.xml`. If this file does not exist, it will be created automatically when you use a Maps component for the first time. To change the configuration, see [Configuring the DataMiner Maps host servers](xref:Configuring_the_DataMiner_Maps_host_servers).
-> - This component currently only supports **Google Maps** ("gmaps") as the [Maps provider](xref:Configuring_the_DataMiner_Maps_host_servers).
+With this component, you can:
+
+- Track both real-time and [historical location data](#using-the-map-for-history-tracking) to monitor movements or analyze trends.
+
+- [Select markers or lines](#using-the-map-as-a-selector) on the map to access more details.
+
+- See your data in a way that makes sense, thanks to a range of [customization options](#customizing-markers).
+
+- Toggle [extra layers on demand](#displaying-extra-layers) to add context without cluttering the map.
+
+## Prerequisites
+
+- To display the map, the machine running the browser must have internet access.
+
+- To use the maps component, the host servers for DataMiner Maps have to be configured in the file `C:\Skyline DataMiner\Maps\ServerConfig.xml`. If this file does not exist, it will be created automatically when you use a maps component for the first time. To change the configuration, see [Configuring the DataMiner Maps host servers](xref:Configuring_the_DataMiner_Maps_host_servers).
+
+## Supported data types
+
+The maps component is used to display the results of queries in a map format. It should therefore **always be configured with [query data input](xref:Query_Data)**.
+
+Each row in a query corresponds to a marker or line on the map.
+
+## Interacting with the maps component
+
+In read mode, you can interact with the maps component in several ways to explore and customize the map view.
```

Comment:
What is the "Read mode" exactly? Is that when not editing the dashboard/app? If so, I wouldn't even mention it.